### ISSUE TITLE
Feature request: top link to switch between AMP and desktop

### ISSUE THREAD

**Issue by AlekzNet** [rickey29/pwamp-mybb18#1]
Great idea! The plugin works fine! 

To make it even more useful and convenient, could you please add a link at the top of the page (or at both top and bottom) to be able switch to the desktop version without adding [&?]desktop to the URL?

**Comment by rickey29** [comment]
Thanks for getting in touch.

I will do some improvement here, since a lot of people complain about this issue.

I will not do it in a short time, because I am doing something important -- right now, I convert every theme into AMP style of MyBB default theme; I will improve to convert every theme into AMP style of that theme, so the outlook of websites will almost not change.  I am busy at this feature now.

I will implement the mobile/desktop switching like website cookie consent banner (the website uses cookie, will write data to and read data from your device......) within the notification, there will be a link, when users click it, they can switch between the two versions.

**Comment by AlekzNet** [comment]
Excellent! Looking forward to these features!

**Comment by rickey29** [comment]
Hi AlekzNet,

I update the code: v2.0.0 .  Let's see if this is what you want.  Any question, feel free to let me know.

**Comment by AlekzNet** [comment]
Yes, I see the switch! However, the site design looks broken - I still see the mybb tables. Can be the multiple caching. Checking....

**Comment by rickey29** [comment]
Sorry for the inconvenience.  Could you send me your website URL by email or PM, so I can do more checking?  You can find my email at my website: flexplat.com; my user Id at MyBB community is: rickey29 .
